force-app/main/default/lwc/registrationForm/registrationForm.html

```diff
@@ -0,0 +1,30 @@
+<template>
+    <lightning-card if:false={validMeetup} title="Invalid Meetup" class="slds-card__body_inner">
+        <p>This meetup can't be found, or doesn't exist. Please verify that you have the correct registration code URL.</p>
+    </lightning-card>
+    <lightning-card if:false={registrationAllowed} title="Meetup Registration Unavailable" class="slds-card__body_inner">
+        <p>This meetup can not currently be registered for.</p>
+    </lightning-card>
+    <lightning-card if:true={showForm} title={meetupName} class="slds-card__body_inner">
+        <lightning-record-edit-form 
+            object-api-name="MeetupRegistration__c" 
+            onsuccess={handleSuccess}
+            onsubmit={handleSubmit}
+            onerror={handleError}
+        >
+            <p>To register for this meetup, fill out and submit the form below.</p>
+            <!-- Reduce form complexity by hiding this lookup, and just displaying the meetup name in card title.-->
+            <lightning-input-field field-name="Meetup__c" disabled="true" value={meetupId} class="slds-hide"></lightning-input-field>
```

Comment:
I wasn't 100% sure on if the individuals should be able to self-select their meetup, so this is a read-only hidden field on the page that gets populated via the url param lookup. It can easily be updated to allow users to self select in the future.